subprojects/core/src/main/java/org/gradle/groovy/scripts/internal/DefaultScriptSourceHasher.java

```diff
@@ -37,22 +37,34 @@ public DefaultScriptSourceHasher(FileHasher fileHasher) {
     }
 
     @Override
-    public HashCode hash(ScriptSource scriptSource) {
+    public HashCode
+    hash(ScriptSource scriptSource) {
```

Comment:
This line break doesn't look like it was intended, was it?

Reply:
Done